include/no_os_print_log.h

```diff
@@ -42,6 +42,12 @@
 
 #include <stdio.h>
 
+#ifdef XILINX_PLATFORM
+#if defined(PRINT_TIME) && !defined(_XPARAMETERS_PS_H_)
+#error The processor time is not available. PRINT_TIME cannot be used.
```

Comment:
On a second thought, I think there are 2 better options:
- drop this commit entirely
- or put a warning in xilinx_delay.c something like below
```
#ifdef _XPARAMETERS_PS_H_
#include "xtime_l.h"
#elif defined(PRINT_TIME)
#warning Time information is not available on Microblaze.
#endif
```

I think I prefer no1 point above, silently returning a time of 0.000000.